docset/winserver2022-ps/bitstransfer/Set-BitsTransfer.md

```diff
@@ -202,6 +322,94 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Dynamic
+Indicates that the transfer uses the dynamic setting.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -HttpMethod
+Specifies a method for the transfer other than the default method GET.
+If you specify GET, the parameter has no effect.
+
+If you specify a method, the job takes foreground priority, which can't be changed.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: hm
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -MaxDownloadTime
+Specifies the maximum time, in seconds, for transferring the files in a job.
+The default is 7,776,000 seconds or 90 days.
+
+```yaml
+Type: Int32
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -NotifyCmdLine
+Specifies a program to run after the job finishes or encounters an error.
+The program runs in the current context.
+
+Specify the program name and any parameters as an array of strings.
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -NotifyFlags
+Specifies the type of event notification you want to receive, such as job transferred events.
+Valid values are:
```

Comment:
Is option 3 an actual value or is it just 1|2? Also should we mention that 3 is the default?

Reply:
@davinci26 - This is where I got the values for this:

https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/bitsadmin-setnotifyflags

I can't tell anything else about 3 from the source.

I'll add a note about the default value.

Reply:
I am under the impression that the option is three is not an actual enum value just the or statement of 1 and 2. Based on that we could probably not mention it since it comes as a combination of 1 and 2

Reply:
OK - I'll do it that way -